openassessment/xblock/validation.py

```diff
@@ -58,27 +62,37 @@ def _is_valid_assessment_sequence(assessments):
         bool
 
     """
-    valid_sequences = [
-        ['self-assessment'],
-        ['peer-assessment'],
-        ['peer-assessment', 'self-assessment'],
-        ['self-assessment', 'peer-assessment'],
-        ['student-training', 'peer-assessment'],
-        ['student-training', 'peer-assessment', 'self-assessment'],
-        ['student-training', 'self-assessment', 'peer-assessment'],
-        ['example-based-assessment'],
-        ['example-based-assessment', 'self-assessment'],
-        ['example-based-assessment', 'peer-assessment'],
-        ['example-based-assessment', 'peer-assessment', 'self-assessment'],
-        ['example-based-assessment', 'self-assessment', 'peer-assessment'],
-        ['example-based-assessment', 'student-training', 'peer-assessment'],
-        ['example-based-assessment', 'student-training', 'peer-assessment', 'self-assessment'],
-        ['example-based-assessment', 'student-training', 'self-assessment', 'peer-assessment'],
-    ]
```

Comment:
Does this method not have a test?  It seems like it would warrant one.  Also, the above seems like it could be used in the test of sequences to determine if it does pass.

In the original comment, it read "...do not allow peer-assessment before self-assessment", but the valid_sequences allow for this.  Which is correct, the code or the comment?